Source/WebKit/WebProcess/WebPage/WebPage.h

```diff
@@ -2888,6 +2892,8 @@ class WebPage : public API::ObjectImpl<API::Object::Type::BundlePage>, public IP
     std::unique_ptr<WebCore::NowPlayingMetadataObserver> m_nowPlayingMetadataObserver;
 
     mutable RefPtr<Logger> m_logger;
+
+    std::optional<WebCore::LoginStatus> m_lastPasswordAutoFill;
```

Comment:
`std::optional<WebCore::LoginStatus> m_lastAuthentication` already exists on `Page` - we should be able to use that and remove this.